public/scripts/controllers/camp_manage.js

```diff
@@ -44,6 +44,18 @@ app.controller("manageCampsController", function ($scope, $http, $filter) {
         }
     }
 
+    // update the camp pre sale quota
+    $scope.updatePreSaleQuota = (camp_id,quota) => {
+        confirm('Confirm new quota to: ' + quota);
+        $.post('/camps/' + camp_id + '/updatePreSaleQuota', { quota: quota })
+           .success(function(response) {
+            window.location.reload();
```

Comment:
You should use the $window service here, which will also require injecting it into the controller.
Using the window object directly is bad form for several reasons:
1. Even if the API is different between browsers, Angular will abstract that from you
2. It's injected as a dependancy so it can be mocked for tests or replaced easily
3. Can include some Angular specific hacks and optimizations

https://docs.angularjs.org/api/ng/service/$window